shopify_app.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency("byebug")
   s.add_development_dependency("minitest")
   s.add_development_dependency("mocha")
+  s.add_development_dependency("nokogiri")
```

Comment:
`nokogiri` was already in the `Gemfile.lock` as a transative dependency, but now that I'm using it in a test I want to make it explicit.